src/include/reconcile.h

```diff
@@ -227,6 +227,14 @@ struct __wt_reconcile {
     size_t supd_allocated;
     size_t supd_memsize; /* Size of saved update structures */
 
+    /*
+     * History store update to be deleted list. While reviewing updates for each page, we save
```

Comment:
I would reword with:
"List of history store updates to be deleted. While reviewing updates for each page, we save them here and delete them after we have built the disk image.".